### PR TITLE
Add quality to heightmap

### PR DESCRIPTION
**Pull request by YoheiKakiuchi**

### CODE REVIEW

File: doc/jsk_perception/nodes/colorize_float_image.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ Colorize float image with heatmap gradation.
...
-  Float(32FC1) image.
+  Float(32FC[1-4]) image.
 ## Publishing Topic
 * `~output` (`sensor_msgs/Image`)
 
   RGB8 image with heatmap gradation.
+## Parameters
+* `~use_channel` (int, default: -1)
+
+  Choose channel to colorize. If value is default, first channel is selected.
```
Comment (wkentaro) [inline doc/jsk_perception/nodes/colorize_float_image.md]: The default can be `0`? Maybe `~channel` is better?

Comment (YoheiKakiuchi) [inline doc/jsk_perception/nodes/colorize_float_image.md]: fixed

Comment (wkentaro) [inline doc/jsk_perception/nodes/colorize_float_image.md]: Thank you.

Comment (wkentaro) [inline doc/jsk_perception/nodes/colorize_float_image.md]: Nice feature of colorize_float_image to select the channel to colorize.
FYI, https://github.com/ros-perception/vision_opencv/pull/150 will expand the feature of `CV_32FC[1-4]` to `CV_32FC[1-N]`.

**Pull request merged**